images/bgt/Dockerfile

```diff
@@ -20,11 +20,12 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends -qqy \
      && rm -rf /tmp/* \
         /var/tmp/* \
         /var/cache/apt/* \
-    && update-ca-certificates \
     && git clone -b ${VERSION} https://github.com/lh3/bgt.git \
     && cd bgt \
     && make \
-    && cp bgt /usr/local/bin/ \
-    && chmod a+x /usr/local/bin/bgt \
-    && cd .. \
-    && rm -rf bgt
\ No newline at end of file
+    && chmod a+x bgt
+
+
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+COPY --from=0 /bgt/bgt /usr/local/bin/bgt
```

Comment:
You can also name stages to make this a bit more readable.

Reply:
Agreed all over, I think this made for a decent template because it's an isolated tool. That will likely apply to a couple of other images, but not many. The Google SDK is going to be quite limiting, but I'm still planning to play around with a different base image e.g. for AIP - there's no reason for that to be a 2.6GB image 😱